src/theme.ts

```diff
@@ -36,5 +21,16 @@ export const theme = createTheme({
 	},
 	typography: {
 		fontFamily: ['"Baloo 2"', 'Roboto'].join(', ')
+	},
+
+	// Overriding component style for the whole app like this:
+	components: {
```

Comment:
Tohle jsem sem přidal prozatím ze dvou důvodů:

- ukazuje to, jak jde stylovat komponenty globálně, tzn. je to potenciální alternativa komponent ve složce `components/forms/generic`, co jsem vytvořil
- je to aktuálně jediný způsob, jak aspoň nějak stylovat label u radiobuttonu (varianta 1)